### ISSUE TITLE
Carret position in masked input

### ISSUE THREAD

**Issue by genpri** [IgniteUI/igniteui-angular#6012]
When users clicks on masked input they expect to be able to write right away but at the current state more steps(click/keyboard navigation) are needed before starting to input data. It is mentioned also here #4249 but after writing there and not receiving any answer looks like that thread is dead, so opening new issue in hope for some answer.

## Describe the solution you'd like
carret automatically at first editable position so the user can start input data right away

## Describe alternatives you've considered
tried palying with focus pipe but just couldnt make it work as per the desired behavior

## Additional context
can see there are some hidden methods in igxMask that handle carret positioning. Are safe to use?Will you open them any time soon?

**Comment by Lipata** [comment]
Hi, @genpri 

You can use the [IgxTextSelection](https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/latest/classes/igxtextselectiondirective.html) directive and it will select all the text when the input is clicked. Please, this [demo](https://stackblitz.com/edit/mask-select-all), I hope it  is suitable for your case.

**Comment by genpri** [comment]
Thank you @Lipata for your fast response.
Your solution is not exactly what we were looking for but plays well with our requirements so will use it for now.
Closing this request.
